### DISCUSSION TITLE
Next Gen UI/UX Pro Tips

### DISCUSSION THREAD

**Discussion by emily-simon** [FlexDeploy-Community/community#20] [Show and tell]
You've probably noticed some of the major global UX enhancements that have come in the last few releases, like saved queries, app search, favorites, recents, and tags. I wanted to take a minute to share some of my favorite app features that haven't gotten as much of the spotlight, but definitely deserve to be mentioned. You can take advantage of these on any screens that have been migrated to the next gen UI/UX. 

- Keyboard shortcuts - Work faster throughout FlexDeploy with customizable keyboard shortcuts. My most used are '/' for global search, 'ctrl+shift+f' to access favorites, 'ctrl+s' to save your current page, and 'ctrl+k' for searching on the current page. You can view all of the shortcuts and also rebind the key mappings from your profile.
![image](https://github.com/FlexDeploy-Community/community/assets/53582991/60daa771-228c-4b78-890d-8995b009f8d6)

- Navigation defaults - Set custom navigation behavior for various tabs from Profile -> Settings to customize things like your default projects tab (all, favorites, or recent) and default package tab (files or execution).

- Multiselect - Many table and list views now have the capability to easily select multiple items and perform actions against them. Right click anywhere on a row to view actions, or open the selected menu above the list or table. Also supported are common keyboard shortcuts for selection like 'shift+click' and 'ctrl+a'.
![image](https://github.com/FlexDeploy-Community/community/assets/53582991/df63452d-955b-41b5-89d4-6624708f10db)

- Context-based documentation links - If you need a quick reference or general guidance for what you're working on, open the relevant FlexDeploy documentation for that page by clicking the **?** icon on the page or pressing 'ctrl+shift+h'.

- Search for pages - You might already be using the global search bar to search for things like projects and work items, but did you know you can also search for pages? Avoid having to navigate through the app menu and simply search for the page you're looking for.
![image](https://github.com/FlexDeploy-Community/community/assets/53582991/0710d214-f84e-4953-a116-ba84c48e0c14)

Are there any features I missed that you're enjoying in the next gen UI? Feel free to share in the comments!

**Comment by erickjones2** [comment]
Another one is quite prevalent in the screen shots.  Dark mode for those, like Emily, who enjoy applications a bit easier on the eyes!  Thanks for sharing!
